doc/locale/ja/LC_MESSAGES/admin-guide/storage/index.en.po

```diff
@@ -242,14 +242,14 @@ msgstr "RAM キャッシュのサイズの変更"
 msgid ""
 "Traffic Server provides a dedicated RAM cache for fast retrieval of popular "
 "small objects. The default RAM cache size is automatically calculated based "
-"on the number and size of the :term:`cache partitions <cache partition>` "
+"on the number and size of the ``cache partitions <cache partition>`` "
 "you have configured. If you've partitioned your cache according to protocol "
 "and/or hosts, then the size of the RAM cache for each partition is "
 "proportional to the size of that partition."
 msgstr ""
 "Traffic Server は人気のある小さなオブジェクトを速く取得するために専用の RAM "
-"キャッシュを提供します。デフォルトの RAM キャッシュサイズは設定した :term:`"
-"キャッシュパーティション` の数とサイズをもとに自動的に計算されます。キャッ"
+"キャッシュを提供します。デフォルトの RAM キャッシュサイズは設定した ``"
+"キャッシュパーティション`` の数とサイズをもとに自動的に計算されます。キャッ"
```

Comment:
I feel bad about this one. I couldn't figure out to how fix the `:term:` for Japanese. Thank you for trying to help, @maskit , but it still failed with your patch. This quiets the warning, but removes the `:term:` link functionality.

@maskit : if you feel motivated, can you look at this? My lack of Japanese knowledge got in the way. If we don't care much, we can go forward with my patch. It at least eliminates the warning. :)

Reply:
This can be reproduced locally like so:

```sh
cmake -B docbuild -DDOC_LANG:STRING=ja -DENABLE_DOCS=ON
cmake --build docbuild --target generate_docs -j10
```

Reply:
If the change on glossary.en.po doesn't work, I think it's a limitation... On glossary.en.rst, "cache partition" is not only a visible string but also an anchor name. I guess Sphinx is confused by that. I don't mind removing `:term:`.